go.sum

```diff
@@ -11,6 +11,7 @@ cloud.google.com/go/firestore v1.1.0/go.mod h1:ulACoGHTpvq5r8rxGJ4ddJZBZqakUQqCl
 cloud.google.com/go/pubsub v1.0.1/go.mod h1:R0Gpsv3s54REJCy4fxDixWD93lHJMoZTyQ2kNxGRt3I=
 cloud.google.com/go/storage v1.0.0/go.mod h1:IhtSnM/ZTZV8YYJWCY8RULGVqBDmpoyjwiyrjsg+URw=
```

Comment:
`go mod tidy` should clean up the changes below

Reply:
I did it but its still here

Reply:
if its critical, I could remove it manually

Reply:
It's alright. I'll check it out after we merge to dev. Maybe these changes should be here